diff --git a/node_modules/three/build/three.cjs b/node_modules/three/build/three.cjs
index f61a72d..b7498a5 100644
--- a/node_modules/three/build/three.cjs
+++ b/node_modules/three/build/three.cjs
@@ -19314,7 +19314,7 @@ function getEncodingComponents( colorSpace ) {
 function getShaderErrors( gl, shader, type ) {
 
 	const status = gl.getShaderParameter( shader, gl.COMPILE_STATUS );
-	const errors = gl.getShaderInfoLog( shader ).trim();
+	const errors = (gl.getShaderInfoLog( shader) || "").trim();
 
 	if ( status && errors === '' ) return '';
 
@@ -20138,9 +20138,9 @@ function WebGLProgram( renderer, cacheKey, parameters, bindingStates ) {
 		// check for link errors
 		if ( renderer.debug.checkShaderErrors ) {
 
-			const programLog = gl.getProgramInfoLog( program ).trim();
-			const vertexLog = gl.getShaderInfoLog( glVertexShader ).trim();
-			const fragmentLog = gl.getShaderInfoLog( glFragmentShader ).trim();
+			const programLog = (gl.getProgramInfoLog(program) || '').trim();
+			const vertexLog = (gl.getShaderInfoLog(glVertexShader) || '').trim();
+			const fragmentLog = (gl.getShaderInfoLog(glFragmentShader) || '').trim();
 
 			let runnable = true;
 			let haveDiagnostics = true;
